### DISCUSSION TITLE
Border outside of the tab

### DISCUSSION THREAD

**Discussion by asmsaiff** [black7375/Firefox-UI-Fix#443] [Q&A] [answered]
Here are a border outside of the tab design and there has a small rounded corder of the tab. How can I get rid of theme like second picture ? 

![2022-07-28_08-50](https://user-images.githubusercontent.com/75557392/181409807-5e35308e-7052-41b0-a754-0b7400b78113.png)

I want like that -

![image](https://user-images.githubusercontent.com/75557392/181410072-e2a4a1af-9a65-4c01-9218-a6d67da4d897.png)

**Comment by black7375** [comment, marked as answer]
You can use `photon-style`'s setting.
https://github.com/black7375/Firefox-UI-Fix/blob/db9962892e13afe992474f2d4e2ea1028819c62a/user.js#L32-L57

**Reply by asmsaiff** [reply]
Thank you so much @black7375 
It works.
